crates/engine/LICENSE

```diff
@@ -0,0 +1,203 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
This should become a one-line soft-link to the crate-root `LICENSE` file as is done by all the other crates.